### PR TITLE
 Feature: Implement Kitsu WebApp

### PR DESCRIPTION
**Pull request by kit0-0**
## Description

This pull request implements the Kitsu WebApp project based on the provided requirements.

## Changes Made

- Integrated the Kitsu API to fetch category data.
- Designed the UI following the specified color, typography, and layout guidelines.
- Implemented the home page to display categories and numeric values.
- Developed the details page to show detailed data for a selected category.
- Created Redux store for managing API data and implemented actions and reducers.
- Implemented React Router for navigation between pages.
- Developed unit tests.
- Deployed the app online using gh-pages.

## Checklist

- [x] Implemented API integration for fetching data.
- [x] Designed UI following guidelines.
- [x] Implemented home page with category list and numeric values.
- [x] Developed details page.
- [x] Created Redux store, actions, and reducers.
- [x] Implemented React Router for navigation.
- [x] Created unit tests for Redux functions.
- [x] Deployed app online.

## Deployment

The app has been successfully deployed and can be accessed at [App Link](https://kit0-0.github.io/react-metrics-anime/).

## Credits

- Design inspiration by Nelson Sakwa.

### CODE REVIEW

File: src/__tests__/animeSlice.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import axios from 'axios';
+import configureMockStore from 'redux-mock-store';
```
Comment (Strangeal) [inline src/__tests__/animeSlice.test.js:2]: - [ ] Great job so far on building your capstone project. However, While reviewing your test, I noticed one of your tests is failing due to a `module` that can't be found. Believe is full details of the failing test details. Kindly take a look at it and please make sure to fix it.
Screenshot 2023-08-07 at 22 09 43

File: README.md [hunk 2]
```diff
@@ -1,70 +1,185 @@
...
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+<!-- LIVE DEMO -->
 
-## Learn More
+## 🚀 Demo <a name="live-demo"></a>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- [🔗 Live Demo Link](https://kit0-0.github.io/react-metrics-anime/)
+- [🔗 Presentation link](https://drive.google.com/file/d/1FQD9Mj7v21qBXtykPSL5ooh5xHmrh7q5/view?usp=sharing)
```
Comment (Strangeal) [inline README.md]: - [ ] Excellent work on articulating your project with a high level of clarity and professionalism. I appreciate the effort you've put into your presentation. However, I would like to kindly request that you re-record the video to align with our presentation requirements.

  - One of the specific requirements outlined is to highlight a particularly significant piece of code that you take pride in. Regrettably, this aspect was not fully met in your current video presentation.

   - To ensure that we adhere to the project's standards and enhance its overall quality, I would encourage you to review the detailed project requirements provided below. By addressing this specific aspect, we can ensure that your presentation fully encompasses the intended objectives.
Screenshot 2023-08-07 at 22 35 09

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @kit0-0, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Pixel Perfect Design ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Successfully deployed 🚀 ✅ 
- Used `gh-pages` for deployment ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kit0-0** [comment]
changes made 

- [🔗 Updated Presentation link](https://drive.google.com/file/d/1QMhXqh-Ou1zPplY-MRfc2iDJ5kBDhCYP/view?usp=sharing)
- add redux-mock-store to dependencies

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @kit0-0 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great
-  ✅ All tests are passing
-  ✅ Descriptive README and PR
-  ✅ Linters are working properly 
-  ✅ Excellent video demonstration

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**